CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
 cmake_minimum_required(VERSION 3.19)
+cmake_policy(SET CMP0091 NEW)
+cmake_policy(SET CMP0042 NEW)
```

Comment:
These aren't necessary: the `cmake_minimum_required` above it means "set all policies added in 3.19 or earlier to their NEW behaviour".